File: Iniciante/1067.c
#include <stdio.h>
 
int main() {

    int X, i, aux;

    scanf("%d", &X);
    aux = X;
    X = X - (aux-1);

    for(i=0; i < aux; i++){      
        if(X % 2 != 0) printf("%d\n", X);
        X++;
    }

    return 0;
}

File: Iniciante/1067.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);
        
        int X = scanf.nextInt();
        int aux = X;
        X = X - (aux-1);

        for(int i=0; i < aux; i++){      
            if(X % 2 != 0) System.out.println(X);
            X++;
        }
    }
}